### ISSUE TITLE
new regression related to bind_rows

### ISSUE THREAD

**Issue by jthaman** [jthaman/ciTools#2]
Run the example code in add_ci.lmerMod.R:

```
dat <- lme4::sleepstudy
 # Fit a linear mixed model (random intercept model)
 fit <- lme4::lmer(Reaction ~ Days + (1|Subject), data = lme4::sleepstudy)
 # Get the fitted values for each observation in dat, and
 # append CIs for those fitted values to dat
 add_ci(dat, fit, alpha = 0.5)
```

I'm getting a warning message related to bind_rows:

```
add_ci(dat, fit, alpha = 0.5)
Warning in bind_rows_(x, .id) :
  binding character and factor vector, coercing into character vector
# A tibble: 180 x 6
   Reaction  Days Subject     pred  LCB0.25  UCB0.75
      <dbl> <dbl>  <fctr>    <dbl>    <dbl>    <dbl>
 1 249.5600     0     308 292.1888 283.0011 301.3766
 2 258.7047     1     308 302.6561 293.5979 311.7143
 3 250.8006     2     308 313.1234 304.1636 322.0832
 4 321.4398     3     308 323.5907 314.6971 332.4843
 5 356.8519     4     308 334.0580 325.1976 342.9183
 6 414.6901     5     308 344.5252 335.6649 353.3856
 7 382.2038     6     308 354.9925 346.0989 363.8861
 8 290.1486     7     308 365.4598 356.5000 374.4196
 9 430.5853     8     308 375.9271 366.8689 384.9853
10 466.3535     9     308 386.3944 377.2066 395.5821
# ... with 170 more rows
```

This has to do with some code in the helper functions file. If it's harmless we can just wrap the code in suppressWarnings(), but I'm not convinced it's entirely harmless.

**Comment by jthaman** [comment]
cc @matthewravery

**Comment by jthaman** [comment]
Fixed since 7ac8e04
